makedeb.sh

```diff
@@ -233,8 +242,9 @@ function build_minion()
     echo
     echo "Version: " $VERSION
     echo "Release: " $RELEASE
+    echo "OPA VERSION: " $OPA_VERSION
     echo
-
+    sed -i "s/OPA_VERSION/$OPA_VERSION/g" $TOPDIR/opennms-assemblies/minion/src/main/filtered/debian/control
```

Comment:
This, especially, since it's in the "filtered" directory, could just have `${opennmsApiVersion}` in it directly, it should get translated at build time.